spec/passaporte_web/identity_spec.rb

```diff
@@ -231,11 +231,11 @@
     # REGRSSION
     it "should return the is_active and id_token fields" do
       identity = described_class.authenticate('mobileteste269@mailinator.com', 'vivalasvegas')
-      identity.is_active.should be_true
+      expect(identity.is_active).to be_truthy
       identity.id_token.should == '9864ec27fb4fd866f6fad5bc041d0363d0bc0fd2945858a1'
     end
     it "should return false if the password is wrong for the given email" do
-      described_class.authenticate('teste@teste.com', 'wrong password').should be_false
+      expect(described_class.authenticate('teste@teste.com', 'wrong password')).to be_falsy
```

Comment:
Line is too long. [91/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.